.ci/test_r_package_windows.ps1

```diff
@@ -36,12 +36,14 @@ if ($env:R_MAJOR_VERSION -eq "3") {
   # Rtools 3.x has to be installed at C:\Rtools\
   #     * https://stackoverflow.com/a/46619260/3986677
   $RTOOLS_INSTALL_PATH = "C:\Rtools"
-  $env:RTOOLS_MINGW_BIN = "$RTOOLS_INSTALL_PATH/mingw_64/bin"
+  $env:RTOOLS_MINGW_BIN = "$RTOOLS_INSTALL_PATH\mingw_64\bin"
+  $env:RTOOLS_PATHS = "$RTOOLS_INSTALL_PATH\bin;" + "$env:RTOOLS_MINGW_BIN;"
```

Comment:
Looks like `$env:RTOOLS_PATHS` is only used for `PATH` enhancement. I think we can add `$env:RTOOLS_MINGW_BIN` directly to `PATH` without proxy `$env:RTOOLS_PATHS` strings storage. And rename `$env:RTOOLS_PATHS` to `$env:RTOOLS_BIN` or something like that. I believe these changes will make the scrip clearer.

Reply:
I don't think that's any better or worse. But since you have a preference and I don't, I'll change it to that.

Reply:
updated in https://github.com/microsoft/LightGBM/pull/3558/commits/16da5c0539caf26942015995a676508f69096e67

Reply:
Thanks!